.gitignore

```diff
@@ -6,3 +6,4 @@
 /laminas-mkdoc-theme/
 /phpunit.xml
 /vendor/
+/.idea/
```

Comment:
please revert this for consistency, you can register your own in your system's global .gitignore.